service/controller/v1/prometheus/scrapeconfig_test.go

```diff
@@ -127,6 +128,7 @@ func Test_Prometheus_GetScrapeConfigs(t *testing.T) {
 		{
 			services:             nil,
 			certificateDirectory: "/certs",
+			provider:             "aws-test",
```

Comment:
This name can be changed if someone cares. It was used because I tested this originally on an aws installation and wanted to differentiate between these newly-applied labels and any existing labels without hiding that the metric came from an aws test installation